### DISCUSSION TITLE
[suggestion] Personal Best Record - Discord

### DISCUSSION THREAD

**Discussion by simpleman21** [runelite/runelite#15192] [Ideas]
Discord webhook integration that logs your personal best records for all bosses, mini-games, etc, into a discord channel with a screenshot (optional toggle).

**Comment by iProdigy** [comment]
[Dink](https://runelite.net/plugin-hub/show/dink) supports sending a notification to discord channels (with optional screenshots) upon personal best records for defeating bosses.
